osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -121,12 +120,13 @@ protected RsaEncryptionService(final KeyPair keyPair) {
      *            the path to a PKCS#8 encoded private key file, decryption will
      *            be disabled if this is blank.
      */
-    public RsaEncryptionService(final String rsaPublicKeyPath, final String rsaPrivateKeyPath) {
+    public RsaEncryptionService(final String rsaPublicKeyPath, final String rsaPrivateKeyPath)
+            throws EncrypterException {
         this.keyPair = this.createKeyPair(rsaPublicKeyPath, rsaPrivateKeyPath);
     }
 
     @PostConstruct
-    private void initEncryption() {
+    private void initEncryption() throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)